src/Core/Enums/RuleId.cs

```diff
@@ -111,7 +113,7 @@ public enum RuleId
         ControlShouldNotSupportTablePattern,
         ControlShouldNotSupportTogglePattern,
         ControlShouldNotSupportValuePattern,
-        ControlShouldNotSupportWindowPattern,
+ControlShouldNotSupportWindowPattern,
```

Comment:
Indent changed (presumably unintentionally)
```suggestion
        ControlShouldNotSupportWindowPattern,
```